src/app/Quiz.js

```diff
@@ -0,0 +1,11 @@
+
+export class Quiz {
+  constructor() {
+    this.people = [];
+    this.vehicles = [];
+    this.starships = [];
+  }
+  set data(data) {
+    this.data.length < 1 && this[data].push(data);
```

Comment:
tego nie kumam, co to ma robić?

Reply:
sprawdza czy dane zostaly juz wprowadzone , jezeli nie to wprowadza  je, jezeli tak, to nic nie robi. Wprowadze jeszcze zabezpieczenie aby nie wysylalo zapytania http jesli dane juz sie znajduja w obiekcie.

Reply:
hm, nie widzę w logach różnicy pomiędzy działaniem z tym setterem i bez niego

Reply:
każdorazowo przy fetchu tworzony jest nowy obiekt i tyle, ale no może czegoś nie kumam

Reply:
debugger też się tu nie zatrzymuję, nie czaję kiedy odpalany jest ten setter

Reply:
Masz racje ten setters w tym momencie jest useless. W ostatnim commit zaproponowalem zmiane.